### ISSUE TITLE
Argparse options for EmailAccounts and Derivatives should be based on what's actually available

### ISSUE THREAD

**Issue by pobocks** [UAlbanyArchives/mailbag#76]
As a user of mailbag, I would like the --help and GUI to accurately reflect the available options for each.  

Recommendation: set the `choices` argument to argparse to the keys of the registry variable for each.

**Comment by gwiedeman** [comment]
#77 addressed this, but I might leave this open as Gooey doesn't allow selecting multiple `choices` for `nargs='+'` so that still needs to be addressed.

**Comment by pobocks** [comment]
Makes sense.  I poked a little at what it would take to make a PR and it's
gonna take some design and a pretty thorough knowledge of wxwidgets.

**Comment by gwiedeman** [comment]
Addressed (if a bit haphazardly) in [fix_gui_args-76 branch](https://github.com/UAlbanyArchives/mailbag/tree/fix_gui_args-76).

Getting an error running from the GUI with the `mailbagit-gui` command:
```
c:\users\greg\appdata\local\programs\python\python37-32\python.exe: can't open file 'C:\Users\Greg\.virtualenvs\mailbag-_Z7D2G4R\Scripts\mailbagit-gui': [Errno 2] No such file or directory
```

However, it runs just fine from a `mailbagit-gui.py` file that just calls `mailbagit.gui()` ¯\_(ツ)_/¯

**Comment by gwiedeman** [comment]
There is a `mailbagit-gui.exe` file in the directory its looking in.
